### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by mcsmith1980** [matghazaryan/AMSlideMenu2#43]
Hi,

Great slide menu, really easy to use, however, it fails on the accessibility side of things (try with VoiceOver enabled).

The menus are selectable within the main screen.
A few fixes:
Within AMSlideMenuMainViewController.m
closeLeftMenuAnimated - completion and closeRightMenuAnimated within 
add self.leftMenu.view.hidden=YES;

closeRightMenuAnimated - completion and closeRightMenuAnimated within 
add self.rightMenu.view.hidden=YES;

Within setup add
self.leftMenu.view.hidden=YES;
add self.rightMenu.view.hidden=YES;

Finally, add an accessibility label to your menu buttons.

The only problem this leaves is your main screen elements are selectable within the slide menus. Not quite sure how to fix this at the moment.

Thanks once again for a great menu system.
Matthew.
